server/app.js

```diff
@@ -21,47 +25,65 @@ const server = http.createServer(app);
 
 const io = socketio(server, {
   cors: {
-    origin: "*",
+    origin: '*',
   },
 });
 
-io.on("connection", (socket) => {
-  console.log("connected");
+io.on('connection', (socket) => {
+  console.log('connected');
 });
 
-if (process.env.NODE_ENV === "development") {
-  app.use(logger("dev"));
+if (process.env.NODE_ENV === 'development') {
+  app.use(logger('dev'));
 }
 app.use(json());
 app.use(urlencoded({ extended: false }));
 app.use(cookieParser());
-app.use(express.static(join(__dirname, "public")));
+app.use(express.static(join(__dirname, 'public')));
 
 app.use((req, res, next) => {
   req.io = io;
   next();
 });
 
-app.use("/auth", authRouter);
-app.use("/users", userRouter);
+app.post('/upload-images', upload.array('image'), async (req, res) => {
```

Comment:
Can you please move this route to the routes folder & the implementation to the controllers folder :)